README.md

```diff
@@ -43,6 +43,17 @@ update.
 * Please note that the date shown is the date that the change was released to the Sandbox or Production.
 
 ---
+### 2 February 2024
+
+* With this change, date validation logic has been amended to allow the submission end date to be same as start date in the following API's
```

Comment:
Just to be clear where the change is applied and maybe reference the relevant error codes:

```suggestion
The following fix relating to RULE_TO_DATE_BEFORE_FROM_DATE and RULE_END_DATE_BEFORE_START_DATE validation errors has been applied to Sandbox and Production:

* With this change, date validation logic has been loosened to allow the submission end date to be same as the start date in the following APIs
```